packages/thirdweb/src/extensions/erc7702/account/types.ts

```diff
@@ -0,0 +1,74 @@
+/* ────────────────────────────────
+   Input types
+   ──────────────────────────────── */
+
+/* ---------- UsageLimit ---------- */
+interface UsageLimitInput {
+  limitType: number;
+  limit: bigint;
+  period: bigint;
+}
+
+/* ---------- Constraint ---------- */
+interface ConstraintInput {
+  condition: number;
```

Comment:
should this be an enum? how do i know what this number maps to. same for limitType